lib/paths.cf

```diff
@@ -360,7 +360,7 @@ bundle common paths
       "path[printf]"        string => "/usr/bin/printf";
       "path[restorecon]"    string => "/sbin/restorecon";
       "path[sed]"           string => "/bin/sed";
-      "path[semodule]"      string => "/usr/sbin/semodule";
```

Comment:
Shouldn't we keep semodule in paths, just add semanage?

Reply:
will add back in with a follow up. :+1: